public/js/bookshelf.js

```diff
@@ -360,7 +366,7 @@ $(document).ready(function(){
 			console.log("remove book id", book_to_remove_id);
 
 			// remove wanted books from wish list
```

Comment:
Let's make this consistent and change all instances of wish list to what it should be